Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -961,9 +969,11 @@ final class ProductStoreTests: XCTestCase {
         let expectedProductSaleEnd = date(with: "2019-10-27T21:29:50")
         let expectedProductTaxStatus = "taxable"
         let expectedProductTaxClass = "reduced-rate"
+        let expectedDownloadableFileCount = 0
+        let expectedDownloadable = false
 
         network.simulateResponse(requestUrlSuffix: "products/\(expectedProductID)", filename: "product-update")
-        let product = sampleProduct(productID: expectedProductID)
+        let product = sampleProduct(productID: expectedProductID, downloadable: true)
```

Comment:
I'd say that in this specific case you can maintain `downloadable` to `false`.

Reply:
It has not been used there. I did some tests and later removed it. Somehow I missed this one. Thanks for pointing out.